crates/ironrdp-graphics/Cargo.toml

```diff
@@ -11,6 +11,9 @@ authors.workspace = true
 keywords.workspace = true
 categories.workspace = true
 
+[features]
+bench = []
```

Comment:
suggestion: Make it clear that this feature is not part of the public API of the crate.
```suggestion
# Internal (PRIVATE!) features. Do not rely on these whatsoever. They may disappear at anytime.
__bench = []
```

Reply:
Right.. do you think there is a standard for that? _ or x- prefix is fine for me.

Reply:
Prefixing with `__` (two underscores) is what I've seen so far as the de-facto standard.

reqwest is doing that:
https://github.com/seanmonstar/reqwest/blob/6e214133ab45ca2e4de493f13e37bb246d05b71a/Cargo.toml#L87-L99

And we followed this style in sspi-rs:
https://github.com/Devolutions/sspi-rs/blob/00158fb8e1425a131960b08469f26f04bfe74d60/Cargo.toml#L71-L75

stylish:
https://github.com/Nullus157/stylish-rs/blob/20da78181ab037993da3291b2736b5ad8908cec3/Cargo.toml#L36

By the way, features prefixed by `_` (even a single one) are considered private and hidden by docs.rs:
https://github.com/rust-lang/docs.rs/blob/a7f23414a35f951c96a1059a858c99ce36bcfe45/src/db/types.rs#L16

So either `_bench` or `__bench`.

The proper solution is still undergoing the RFC process: https://github.com/rust-lang/rfcs/pull/3487